docs/packetsampling-guide.md

```diff
@@ -0,0 +1,76 @@
+# PacketSampling User Guide
+
+Starting with Antrea v2.0, Antrea supports using PacketSampling for network diagnosis.
+It can capture specified number of packets from real traffic and upload them to a
+supported storage location. Users can create a PacketSampling CRD to trigger
+such actions on the target traffic flow.
+
+<!-- toc -->
```

Comment:
I feel this toc is not very useful since the doc is quite short and has only two sections.